src/activities/multiplication-chart/MultiplicationChart.qml

```diff
@@ -0,0 +1,312 @@
+/* GCompris - multiplication-chart.qml
+ *
+ * Copyright (C) 2016 Varun Kumar <varun13169@iiitd.ac.in>
+ *
+ * Authors:
+ *
+ *   Varun Kumar <varun13169@iiitd.ac.in>
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+
+import "../../core"
+import "multiplication-chart.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Image {
+        id: background
+        fillMode: Image.PreserveAspectCrop
+        source: "qrc:/gcompris/src/activities/multiplication-chart/resource/numbers.svg"
+        sourceSize.width: parent.width
+        sourceSize.height: parent.height
+
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+            property alias repeaterGridRow: repeaterGridRow
+            property alias repeaterGridCol: repeaterGridCol
+            property alias repeater: repeater
+            /*variables*/
+            property int spacing: 5
+            property int rectHeight: wrap.height/25
+            property int rectWidth: wrap.height/25
+            property int rowSelected: 0
+            property int colSelected: 0
+            property int rowQues: 0
+            property int colQues: 0
+        }
+
+        onStart: Activity.start(items, additionalItem)
+        onStop:  Activity.stop()
+
+
+
+        /////////////////////////////////////////////////////
+        Rectangle {
+            id: wrap
+            color: "transparent"
+            opacity: 1
+            height: background.height < background.width ? background.height : background.width
+            width:  background.height < background.width ? background.height : background.width
```

Comment:
Generally height is smaller than width, but this is to cover the case when window is resized and width becomes smaller than height. :)